README.md

```diff
@@ -131,6 +131,45 @@ import { SessionTrace } from '@newrelic/browser-agent/features/session_trace';
 import { Spa } from '@newrelic/browser-agent/features/spa';
 ```
 
+## Deploying a "micro" agent, for use with segmented UI design involving manual implementations only
+
+The examples above use the `Agent` class at their core, which is ideal for most cases as it will automatically detect page-level events across your web application.
+
+Using the `MicroAgent` class, it is possible to skip the "auto" instrumentation phases of the other loader types, and provide a *very small* agent that only responds to user input via the 
+API interfaces.  The `MicroAgent` only supports capturing a `Page View Event` when first instrumented, or features tied to API implementations, which at this time are `PageActions` and `JavaScript Errors`. These events will only be captured by the `MicroAgent` if their relevant API methods are executed.  By not wrapping the page-level globals in the same way as the base `Agent` class, the `MicroAgent` can easily be instantiated multiple times on a single page with low overhead and report to different entities based on the configurations supplied.  This allows for specialized use cases, such as those within micro front-end scenarios or in applications requiring subsets of manually-handled data to be reported to different application entities.
```

Comment:
I reorganized this a bit to highlight the two distinct benefits: size and multiplicity. I think it will be interesting to see how many folks use a micro-agent by itself (loader type micro, agents per page 1).